plankton/devices/__init__.py

```diff
@@ -187,95 +186,3 @@ def _override_data(self, overrides):
                         '\'{}\' of class \'{}\'.'.format(name, type(self).__name__))
 
                 setattr(self, name, val)
-
-
-def is_device(obj):
-    return issubclass(obj, CanProcess) and obj.__module__ not in (
-        'plankton.core.processor', 'plankton.devices')
-
-
-def import_device(device, setup=None, device_package='devices'):
```

Comment:
Similar to a previous comment, it's great to see these removed from here.

Could the remaining `Device` and `StateMachineDevice` classes be moved somewhere else? Could be added to `core/devices.py`. Alternatively, could move them to a `core/device.py` together with `DeviceBase` and rename the remainder of `core/devices.py` to something more general (since it also handles finding interfaces and setups)... `discovery.py` or `registry.py`?

Again, no change necessary for this PR, just a general comment.